pkg/controller/certificates/issuing/issuing_controller.go

```diff
@@ -158,7 +168,7 @@ func NewController(
 
 func (c *controller) ProcessItem(ctx context.Context, key string) error {
 	// Set context deadline for full sync in 10 seconds
-	ctx, cancel := context.WithTimeout(ctx, time.Second*10)
+	ctx, cancel := context.WithTimeout(ctx, c.contextTimeout)
```

Comment:
question: Do you have a reason you're wanting to configure the timeout here? To fix the linked issues I'd expect that we'd only want to configure `pkg/controller/clusterissuers/sync.go` and `pkg/controller/issuers/sync.go`.

Reply:
I'd be +1 on this to address the issue with slow Sectigo endpoints timing out on both cluster issuer setup and orders as well

Reply:
hi sorry for the late response.

@SgtCoDFish the problem with context timeouts is that I got quite confused which one is actually triggered.

So neither the above change nor hardcoding it to lets say 60 seconds helped to mitigate that problem. I gone ahead and looked around what else is using WithTimeout in that controller context but I guess I've just overseen something.

I currently don't have too much time to look further into this issue. I hope I can find some more hours to spent on this in the next few weeks.

I guess it definitely needs a test so its easier to reproduce this problem